### PR TITLE
Enhance: Dockerized inventory-manager

### PR DESCRIPTION
**Pull request by xiaods**
- [x] use docker-compose.overide.yml to override all variables which are in the config files by using ENV variables.

- [x] I also want to be able to tell the docker do I need to run npm run dev or npm run qa or any other npm scripts command.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/ColuLocalNetwork/inventory-manager/blob/master/.github/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by LiorRabin — CHANGES_REQUESTED** [review]
Hello @xiaods, a few changes please:
1. We don't need to use the `.env` file as we already have static configuration files under `/config` folder. All non-static configuration like mongo for example can be a part of the `docker-compose.yml` in the `environment` section.
2. remove `docker-compose.prod.yml` and `docker-compose.test.yml` and add a `docker-compose-inventory-only.yml` or something similar.
3. Why the `OSSEUS_ENV` variable name and not just `ENV` ?
4. Why added `process.env.XXX` on the local configuration files?

**Comment by xiaods** [comment]
@LiorRabin  i have update codebase for your review. please have a look again. thanks .

**Review by LiorRabin — CHANGES_REQUESTED** [review]
@xiaods thanks for the update, I still see you're using `.env` file - is there a way to remove it entirely? even in the `docker-compose.yml` and just use the `environment` property of the yml file?

**Comment by xiaods** [comment]
@LiorRabin yes, remove .env file entirely and update README clearly. please check again.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The inventory manager is an [osseus](https://github.com/colucom/osseus) based se
 
 It provides blockchain reflection of offchain activities on community currencies issued against the [CLN](https://cln.network/).
 
+## Dockerized workflow
+
+Dev: `docker-compose up -d`
+
+Test, QA, Production with appropriate npm command: `cp docker-compose-inventory-only.yml docker-compose.override.yml && docker-compose up -d`
```
Comment (LiorRabin) [inline README.md]: @xiaods why do we need to copy the yml file to `*.override.yml` file?

**Review by LiorRabin — APPROVED** [review]

**Pull request merged**